doc/source/changes/version_0_29.rst.inc

```diff
@@ -111,6 +111,21 @@ New features
     >>> arr = ndtest((3, 3))
     >>> arr.meta.title = 'array for testing'
 
+* made a few useful constants accessible directly from the larray module: nan, inf, pi, e and euler_gamma. Like for
+  any Python functionality, you can choose how to import and use them. For example, for pi:
+
+    >>> from larray import *
+    >>> pi
+    3.141592653589793
+    OR
+    >>> from larray import pi
+    >>> pi
+    3.141592653589793
+    OR
+    >>> import larray as la
+    >>> la.pi
+    3.141592653589793
+
```

Comment:
Is the last example really necessary? None of our current users use `import larray as la` and I guess future hypothetical external users will know.

Reply:
Given that reasoning, the second one is useless too. Honestly, it was just an excuse to show them the proper way to do things because I feel guilty encouraging our users in the "easy-but-dangerous" way.

Reply:
In that case, don't change anything.